tests/rest/test-rest-api.php

```diff
@@ -264,6 +407,11 @@ public function test_rest_api_returns_error_for_classic_content() {
 		$this->expectException( \PHPUnit\Framework\Error\Error::class );
 
 		$request  = new WP_REST_Request( 'GET', sprintf( '/vip-block-data-api/v1/posts/%d/blocks', $post_id ) );
+		$request->set_query_params( array( 
```

Comment:
Is this supposed to be in here?

Reply:
Nope, it's a merge conflict that got resolved incorrectly since the two tests were one after the other. Good catch